### PR TITLE
update bench inis and add small plot

### PR DESCRIPTION
**Pull request by awirb**
* force entrainment and resistance off for some benchmark tests 
* add small plot of distribution of differences for quick plot

### CODE REVIEW

File: avaframe/out3Plot/outQuickPlot.py [hunk 1]
```diff
@@ -122,6 +123,14 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
     ax3.set_aspect('auto')
     ax3.set_xlabel('x [m]')
     ax3.set_title('Difference ref-sim')
+
+    # for difference histogramm - remove dataDiff == 0 values from array
+    dataDiffPlot = dataDiff[dataDiff != 0.0]
+    axin2 = ax3.inset_axes([0.75, 0.1, 0.25, 0.25])
```
Comment (fso42) [inline avaframe/out3Plot/outQuickPlot.py:129]: Is there way to place this automatically? Similiar to what is available for the legend...

**Review by fso42 — APPROVED** [review]
There is just one question in there...

File: avaframe/out3Plot/outQuickPlot.py [hunk 2]
```diff
@@ -71,10 +73,17 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
...
-    diffMin = np.amin(dataDiff)
-    diffMean = np.mean(dataDiff)
+    dataMask = np.ones((ny, nx))
+    for m in range(ny):
+        for k in range(nx):
+            if data1[m,k] == 0.0 and data2[m,k] == 0.0:
+                dataMask[m,k] = np.nan
+
+    dataDiff = data1 - data2
+    dataDiff[np.isnan(dataMask)] = np.nan
```
Comment (matthiasto) [inline avaframe/out3Plot/outQuickPlot.py:83]: dataDiff = data1 - data2
dataDiff = np.where((data1==0 & data2==0), np.nan, dataDiff)
should do the same

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/out3Plot/outQuickPlot.py (hunk 2 above)
Comment (awirb) [reply avaframe/out3Plot/outQuickPlot.py]: I changed this

File: avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini [hunk 3]
```diff
@@ -103,7 +103,7 @@ entDefResistance = 0
 
 #+++++++++++++ Output
 # desired result Parameters
-resType = FD_FV
+resType = FD_FV_pfv
```
Comment (fso42) [inline avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini:106]: Is there a reason to mix upper and lower case letters here?

File: avaframe/runDamBreak.py [hunk 4]
```diff
@@ -25,7 +25,8 @@
 cfgMain['MAIN']['avalancheDir'] = avaDir
 
 # Clean input directory(ies) of old work and output files
-initProj.cleanSingleAvaDir(avaDir, keep=logName)
+#initProj.cleanSingleAvaDir(avaDir, keep=logName)
```
Comment (fso42) [inline avaframe/runDamBreak.py:28]: remove

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/runDamBreak.py (hunk 4 above)
Comment (awirb) [reply avaframe/runDamBreak.py]: done

File: avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini (hunk 3 above)
Comment (awirb) [reply avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini:106]: the non-peak fields use capital letters so far..

**Pull request merged**